themes/mlibrary_new/index.php

```diff
@@ -1,65 +1,55 @@
 <?php 
         set_theme_option('display_header','1');
 	echo head(array('bodyid'=>'home')); 
-	$sidebar_pos=get_theme_option('sidebar_position');
-	$main_add="";
-	$sidebar_add="";
-	if($sidebar_pos=='left'){
-		$main_add=" col-md-push-4";
-		$sidebar_add=" col-md-pull-8";
-		}		
 ?>
 <div class="row">
-    <div class="col-md-8<?php echo $main_add;?>">
-		<?php if ($homepageText = get_theme_option('Homepage Text')): ?>
-			<div id="homepage-text"><p><?php echo $homepageText; ?></p></div>
-		<?php else: ?>
-		<?php echo $this->partial('common/loremize-main.phtml');?>
-		<?php endif; ?>
-
+    <div class="col-xs-12">
+    <div class="row detail-nav">
+    	<div class="col-xs-12 ">
+    		<div class="col-xs-12 col-sm-6">
+    			<ul>
+    			<li class="detail-nav-heading">Browse</li>
+    			<li>Vivamus sagittis lacus vel augue laoreet rutrum faucibus dolor auctor.</li>
+    			<li><a href="<?php echo url('exhibits'); ?>">Browse online exhibits</a></li>
+    			</ul>
+	    	</div>
+	    	<div class="col-xs-12 col-sm-6">
+	    		<ul>
+    			<li class="detail-nav-heading">About</li>
+    			<li>Etiam porta sem malesuada magna mollis euismod.</li>
+    			<li><a href="<?php echo url('about'); ?>">About online exhibits</a></li>
+    			</ul>
+	    	</div>
+    	</div>
+    </div>
+    <div class="col-xs-12 ">
+      <div class="detail-nav-border"><hr></div>
```

Comment:
[The `<hr>` tag's semantics don't mean what it used to](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/hr).

I recommend drawing a horizontal line with border-bottom on detail-nav with appropriate margin-/padding-bottom to get the vertical space you need.